<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1689</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>30</y>
      <width>421</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Вы желаете изменить информацию о больном</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>540</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QPushButton" name="SortByDiagnozButton">
       <property name="text">
        <string>Вывести</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="SortByDateButton_2">
       <property name="text">
        <string>Отсортировать</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Вывести пациентов с 3+ диагнозами:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Отсортировать по убыванию даты поступления:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>100</y>
      <width>401</width>
      <height>424</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="10" column="0">
      <widget class="QLabel" name="ageIntervalLabel_2">
       <property name="text">
        <string>Дата выписки:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="fioLabel_2">
       <property name="text">
        <string>ФИО:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="weightLabel_2">
       <property name="text">
        <string>Дата поступления:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="NumberSpin"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="highLabel_2">
       <property name="text">
        <string>Диагноз </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Number">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Номер больного:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="changeButton_2">
       <property name="text">
        <string>Поменять!</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QRadioButton" name="womanRadioButton_2">
       <property name="text">
        <string>Женский</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QTextEdit" name="updateDiagnoz_4"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="sexLabel_2">
       <property name="text">
        <string>Пол</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="ageLabel_2">
       <property name="text">
        <string>Возраст</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="manRadioButton_2">
       <property name="text">
        <string>Мужской</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="AgeSpin_3"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QTextEdit" name="enterFIO_5"/>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QTextEdit" name="enterAdmissionDate"/>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="DiagnozSpin_2"/>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QTextEdit" name="enterExtractDate"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="addButton_2">
       <property name="text">
        <string>Добавить!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>729</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить больного №:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="personForDeleteNumberSpin">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>720</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1231</width>
      <height>741</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ФИО</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Пол</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Возраст</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Диагноз</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Номер палаты</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата поступления</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата выписки</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>720</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>60</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string> или добавить нового больного?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>YYYY-MM-DD</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1689</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="OpenAction"/>
    <addaction name="SaveAction"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="OpenAction">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="SaveAction">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
